### ISSUE TITLE
Missing spaces when copying CV content

### ISSUE THREAD

**Issue by tzhern** [themagicalmammal/MagicalCV#14]
<img width="959" alt="Screenshot 2023-07-24 at 21 31 53" src="https://github.com/themagicalmammal/Resume/assets/23621760/e7a62555-0e8b-4216-9d55-131760d578d6">

The issue happens when I download the PDF (no issue on the one compiled on Overleaf online), and copy the content from the downloaded PDF. The result is as followed:
```
AUniversityofLeedsgraduatestudentwhoisenrolledintheDataScienceandAnalysisprogramme. Dedicatedtosoftwareengineering,having
twoormoreyearsofexpertise,andspecialisinginfull‑stackwebdevelopment,algorithms,andmachinelearning. SearchingmostlyforMachine
Learning,DataEngineer,andPythonDevelopmentpositions.
```

**Comment by zzmasoud** [comment]
Yes, I confirm this. Actually, I thought It was just me with my system! Because some scanner apps could extract the text correctly.

**Comment by themagicalmammal** [comment]
Sorry for the late reply, I have tried the pdf(s), i didn't find issues I have still updated it, can you try the new cls and let me know if you still have the issue.

**Comment by themagicalmammal** [comment]
Please contact me on telegram regarding this. I can help out!

**Comment by themagicalmammal** [comment]
Closing issue because no bugs reported for a year.
